backend/substrapp/views/filters_utils.py

```diff
@@ -96,8 +98,12 @@ def _get_model_tuple(model):
 
 
 def filter_list(object_type, data, query_params):
-
-    filters = get_filters(query_params)
+    try:
+        filters = get_filters(query_params)
+    except Exception as e:
+        # TODO add better filters parsing to avoid this catch all
+        logging.exception(e)
```

Comment:
I think it would be useful here to log query_params that led to this exception, what do you think?

Reply:
Either in the log or in the BadRequestError raised under.

Reply:
Yes good idea, we should add the query_params to the BadRequestError.
In an ideal world we would have a better parsing of the query args with better validation error.

Reply:
Done, it has been added to the BadRequest response and to the exception log message.